tools/autograd/templates/Functions.cpp

```diff
@@ -48,6 +48,10 @@ void copy_range(variable_list& out, IndexRange range, at::ArrayRef<Tensor> t) {
   std::copy(t.begin(), t.end(), out.begin() + range.first);
 }
 
+std::vector<Tensor> to_tensor_list(const variable_list& variables) {
+  return fmap(variables, [](torch::autograd::Variable v) { return static_cast<Tensor>(v); } );
```

Comment:
Please make it `const torch::autograd::Variable& v`. Right now it incurs an extra temporary refcount increase, which is unnecessary